tools/system_libs.py

```diff
@@ -614,7 +614,6 @@ def create_libcxx():
     return build_libcxx(os.path.join('system', 'lib', 'libcxx'), 'libcxx.a', libcxx_files, ['-Oz', '-Wno-warn-absolute-paths', '-I' + shared.path_from_root('system', 'lib', 'libcxxabi', 'include')], has_noexcept_version=True)
 
   def apply_libcxx(need):
-    assert shared.Settings.QUANTUM_SIZE == 4, 'We do not support libc++ with QUANTUM_SIZE == 1'
     #logging.info('using libcxx turns on CORRECT_* options')
     return True
```

Comment:
Looks like we can totally eliminate all of the apply_ functions now? They all return true.